### PR TITLE
planner: fix point get with PK and UK

### PR DESCRIPTION
**Pull request by jackysp**
### What problem does this PR solve? 
Point get will miss the unique key if the primary key is also in the condition.

### What is changed and how it works?
If check the primary key failed, go to check the unique key.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: planner/core/point_get_plan.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ func tryPointGetPlan(ctx sessionctx.Context, selStmt *ast.SelectStmt) *PointGetP
 	handlePair := findPKHandle(tbl, pairs)
 	if handlePair.value.Kind() != types.KindNull {
 		if len(pairs) != 1 {
-			return nil
+			goto CheckUK
```
Comment (XuHuaiyu) [inline planner/core/point_get_plan.go]: Will it be better to add a comment here?

Comment (erjiaqing) [inline planner/core/point_get_plan.go]: Since we can merge the two if, why `goto` here?

Comment (winkyao) [reply planner/core/point_get_plan.go]: How about wrap 207-226 lines to a function, and call it here, to avoid `goto`

Comment (jackysp) [reply planner/core/point_get_plan.go]: Both OK, we should try the effect first.

**Pull request merged**